package.json

```diff
@@ -7,7 +7,9 @@
     "build": "next build",
     "start": "next start",
     "test": "jest --watch",
-    "test:ci": "jest --ci --passWithNoTests"
+    "test:ci": "jest --ci --passWithNoTests",
+    "local": "docker-compose up -d && npx prisma migrate dev && npx prisma studio",
```

Comment:
I made a script command that when you run the command, it will create a docker container, update the Prisma schema, and open Prisma studio on the browser.

Reply:
I like how you included the Docker command in here, but I think the Prisma migration should not happen on each execution of the server. Schema changes don't happen often and can be a manual process of running the command. What would be good is to include instructions on running migrations in the readme if you haven't already.